backend/climateconnect_api/management/commands/create_test_data.py

```diff
@@ -135,21 +136,31 @@ def create_project_test_data(number_of_rows: int):
         url_slug = name.replace(" ", "")
 
         if not Project.objects.filter(name=name).exists():
+            # Buffer some start date to create more
+            # sense of older projects. This originated out of
+            # fixing a bug around the "timeago" utility, where
+            # timeago would round up to 2 years, when a project
+            # is only 1 year and 1 day old. This should create the
+            # UTC timestamp, e.g. 2020-02-04T01:33:45.276997Z
+            one_year_and_one_day_ago = timezone.now() - timezone.timedelta(days = 366)
+
             project = Project.objects.create(
                 name=name,
                 city="Test {}".format(i),
                 collaborators_welcome=True,
                 country="Germany",
                 short_description="This is a test project.",
-                start_date=timezone.now(),
+                start_date=one_year_and_one_day_ago,
```

Comment:
Hmmmm... Makes sense. Should we do random start_date maybe? Don't think its necessary just a thought.

Reply:
I like that idea for generating more entropy and randomness! We could revisit in a future PR.